src/pages/Home.jsx

```diff
@@ -1,20 +1,22 @@
-import Hero from "../components/home/Hero";
-import About from "../components/home/About";
-import VolunteerBrief from "../components/home/VolunteerBrief";
-import Events from "../components/home/Events";
-import ProjectsBrief from "../components/home/ProjectsBrief";
-import Partners from "../components/home/partners/Partners";
+import Hero from '../components/home/Hero';
+import About from '../components/home/About';
+import VolunteerBrief from '../components/home/VolunteerBrief';
+import Events from '../components/home/Events';
+import ProjectsBrief from '../components/home/ProjectsBrief';
+import Partners from '../components/home/partners/Partners';
+import { Container } from '@mui/material';
```

Comment:
And then add `// Material UI Imports` here. And so far we've been putting these imports above the component ones.